logstash-core/src/main/java/org/logstash/jackson/StreamReadConstraintsUtil.java

```diff
@@ -18,6 +18,14 @@ public class StreamReadConstraintsUtil {
 
     private StreamReadConstraints configuredStreamReadConstraints;
 
+    // Provide default values for Jackson constraints in the case they are
+    // not specified in configuration file.
+    public static final Map<Override, Integer> JACKSON_DEFAULTS = Map.of(
+        Override.MAX_STRING_LENGTH, 200_000_000,
+        Override.MAX_NUMBER_LENGTH, 10_000,
+        Override.MAX_NESTING_DEPTH, 1_000
```

Comment:
CODEREVIEW: I just noticed this is actually not set explicitly https://github.com/elastic/logstash/blob/0eb03ad09893d4efb838d365efc0b6235b711e0a/config/jvm.options#L90  (it was commented out). Should i remove this setting?

Reply:
@edmocosta Is it intended to comment out? I think it is supposed to be uncommented

Reply:
Yes, it's intended to be commented out, at time we added those properties, `1000` was the default value already, and we haven't had any problem with that specific limit that would justify overriding it. 

It seems the Jackson's team is still tuning it, and the new default value was reduced to `500` (https://github.com/FasterXML/jackson-core/pull/1234). They've a good reasoning on why it was reduced (https://github.com/FasterXML/jackson-core/issues/1233), but I think we can be more conservative and sticky to the well tested `1000` (at least for now).  With this new code, it seems uncommenting the property wouldn't make much difference, so I'm fine with whatever you folks think is better.

Reply:
thanks, keeping the comment is good to me.

Reply:
Wow it sounds we lost consistency here as well.
Do we know till which jackson version (and LS version) we had nesting-depth with 1000 and which version decreased to 500 in which LS version? Do we have issues reported for such situation? If so, addressing separately would be good idea considering user stories.

Reply:
Sorry, my comment was very ambiguous after reading it following a break 😅 

My question is *given* we have this commented out *should* I remove the default from being applied. I was not suggesting we remove the example from the options file.

Reply:
> Given we have this commented out should I remove the default from being applied 

IMO, I'd keep it, it's a no-op for now, but knowing that Jackson will reduce this limit in the near future, setting it here would ensure that all defaults values are aligned with Logstash's requirements.